.travis.yml

```diff
@@ -8,7 +8,7 @@ env:
 install:
   - pip install -r requirements.txt
 # command to run tests
-script: nosetests test/tests.py --with-coverage --cover-package=test && coverage report
+script: nosetests test/*.py --with-coverage --cover-package=test && coverage report
```

Comment:
Kindly,Dont tests the tests, test the module files.

Reply:
Please explain

Reply:
instead of `script: nosetests test/tests.py --with-coverage --cover-package=test  && coverage report`
it should be `script: nosetests test/tests.py --with-coverage --cover-package=views.py  && coverage report`
because travis checks on the tests and compares with your functionality to determine if you tested  everything...but when you run the tests it will check on the tests and compares with the same tests file.

Reply:
I have understood your point, Thanks